src/amo/components/Addon/styles.scss

```diff
@@ -193,13 +193,23 @@
   .Addon-details {
     display: grid;
     grid-auto-flow: column dense;
-    grid-gap: 10px;
+    grid-gap: 0 10px;
     grid-template-columns: minmax(300px, 35%) auto;
     margin: 10px 0 0;
 
     .AddonDescription {
       grid-column: 2;
       grid-row: 1 / span 5;
+
+      .Addon--has-more-addons.Addon-persona & {
+        // This span makes sure the left widdget does not move vertically when
+        // there is a long description.
+        grid-row: 2 / span 1000;
```

Comment:
Hah, I did something similar at first but was worried it was too hacky. If you're gonna do this and want things to really line up you're best going for a wild value. Might as well kick it up to `50000`. Seriously 😅